gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -393,6 +397,10 @@ bool Vulkan::replayCreateVkDeviceImpl(Stack* stack, size_val physicalDevice,
 	        xcbInfo.window = info->window;
 	      }
 	      break;
+      case gapir::SurfaceType::Ggp:
```

Comment:
Indentation is off, which is quite confusing.

Reply:
Fixed.

Reply:
Still off compared to L391.